Makefile

```diff
@@ -12,7 +13,8 @@ build:
 .PHONY: serve
 serve: build
 serve:
-	$(WAGI) -c ./modules.toml --log-dir ./logs -e PREVIEW_MODE=$(PREVIEW_MODE) -e SHOW_DEBUG=$(SHOW_DEBUG)
+	rm -rf config/_cache.json
```

Comment:
To enable scenarios where devs do want to use the cache (DISABLE_CACHE=0), we'd not want to run this command, right?  Maybe we could put it into a separate `clean` target and/or optionalize its execution...

Reply:
Ah, yeah -- good call.